hydrolib/tools/ext_old_to_new/converters.py

```diff
@@ -118,6 +160,39 @@ def convert(self, forcing: ExtOldForcing) -> Boundary:
         return new_block
 
 
+def create_convert_inputs(forcing: ExtOldForcing) -> Dict[str, str]:
```

Comment:
Make the function name more descriptive + add docstring (I can reverse engineer that this one is probably shared functionality for initial and parameter fields (i.e., it is intended to produce a dict that can be used to construct an instance of some AbstractSpatialField subclass.)